docs/preview/dev/building/windows.md

```diff
@@ -56,13 +56,13 @@ go build
 collect2.exe: error: ld returned 5 exit status
 ```
 
-GitHub user [vdmitriyev](https://github.com/vdmitriyev) shared instructions for [building the DuckDB Go client on Windows](https://github.com/marcboeker/go-duckdb/issues/4#issuecomment-2176409066):
+GitHub user [vdmitriyev](https://github.com/vdmitriyev) shared instructions for [building the DuckDB Go client on Windows](https://github.com/duckdb/duckdb-go/issues/4#issuecomment-2176409066):
```

Comment:
Here we probably need to keep linking to the (soon to be archived) issue in the marcboeker repository.